test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -1258,67 +1258,63 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 		It("should create pv with zone labels and node affinity", func() {
 			if !eng.ExpandedDefinition.Properties.HasLowPriorityScaleset() {
 				if eng.ExpandedDefinition.Properties.HasZonesForAllAgentPools() {
-					if !common.IsKubernetesVersionGe(eng.ExpandedDefinition.Properties.OrchestratorProfile.OrchestratorVersion, "1.17.0-alpha.1") {
```

Comment:
I wonder if we should restore the if condition for 1.17 + CCM?

Reply:
if not, that is

Reply:
Yes, it should have been testing for version >= 1.16 and `useCloudControllerManager == true`, like the test below on line 1784 does. Fixed.